data/gala.gschema.xml

```diff
@@ -148,7 +147,7 @@
       <summary>Zoom out</summary>
     </key>
     <key type="as" name="expose-windows">
```

Comment:
Do we need this key at all? If not we should change the description of the key to reflect that it's deprecated as of this version and will be removed in the next major version

Reply:
I marked it as deprecated now.